pom.xml

```diff
@@ -7,7 +7,7 @@
     <name>Beeline Java (Parent)</name>
     <artifactId>beeline-parent</artifactId>
     <groupId>io.honeycomb.beeline</groupId>
-    <version>1.1.0</version>
+    <version>1.1.1</version>
```

Comment:
Please revert this back to 1.1.0, we'll do the version bump in a separate PR.

Reply:
As discussed in slack, I have not reverted this change.